packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -193,6 +199,9 @@ export enum ContainerMessageType {
 
 	// Sets the alias of a root data store
 	Alias = "alias",
+
+	// Id allocation op
```

Comment:
A little more information on what this is might be helpful. Maybe, also point to the README?

Reply:
Added some more info here and a link to the IdCompressor README